.github/compatibility-test-matrices/unreleased/incentivized-transfer.json

```diff
@@ -0,0 +1,15 @@
+{
+  "chain-a": ["release-v6.0.x", "release-v5.1.x", "release-v5.0.x", "release-v4.2.x"],
+  "chain-b": ["release-v6.0.x", "release-v5.1.x", "release-v5.0.x", "release-v4.2.x"],
```

Comment:
i guess pending the updates on the SDK we will have to update the 5.1 and 5.0 lines to be 5.2 🙈

Reply:
Maybe we can actually get this merged and then bump the versions once we have the new lines.